static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts

```diff
@@ -62,58 +64,37 @@ const adManagerGroups = [
 
 type Frequency = typeof frequency[number];
 type AdManagerGroup = typeof adManagerGroups[number];
-type ContentType =
-	| 'video'
-	| 'tag'
-	| 'section'
-	| 'network-front'
-	| 'liveblog'
-	| 'interactive'
-	| 'gallery'
-	| 'crossword'
-	| 'audio'
-	| 'article';
-
-export type PageTargeting = PartialWithNulls<{
-	ab: string[];
-	af: 't'; // Ad Free
-	amtgrp: AdManagerGroup;
-	at: string; // Ad Test
-	bl: string[]; // BLog tags
```

Comment:
Why are `bl` and others now removed?

Reply:
Ahh do these live in `SharedTargeting`?

Reply:
yeah exactly

Reply:
+1 to adding all the other types from commercial-core in the same way - or we could export one PageTargeting type from core once everything has moved across.